src/topp/FileInfo.cpp

```diff
@@ -167,7 +167,7 @@ class TOPPFileInfo : public TOPPBase
   }
 
   template <class Map>
```

Comment:
do we need the template at all?

Reply:
we do. Its used for FeatureMaps as well.